frontend/src/features/public-form/components/FormStartPage/FormLogoutButton.tsx

```diff
@@ -0,0 +1,47 @@
+import { BiLogOutCircle } from 'react-icons/bi'
+import { Box } from '@chakra-ui/react'
+
+import { useIsMobile } from '~hooks/useIsMobile'
+import Button from '~components/Button'
+import IconButton from '~components/IconButton'
+
+import { FormBannerLogoProps } from './FormBannerLogo'
+
+export interface FormLogoutButtonProps
+  extends Pick<FormBannerLogoProps, 'loggedInId' | 'colorScheme'> {
+  onLogout?: () => void
+}
+
+export const FormLogoutButton = ({
+  loggedInId,
+  onLogout,
+  colorScheme,
+}: FormLogoutButtonProps): JSX.Element | null => {
+  const isMobile = useIsMobile()
+
+  if (!loggedInId) {
+    return null
+  }
+
+  return (
+    <Box alignSelf="center" justifySelf="flex-end">
+      {isMobile ? (
+        <IconButton
+          colorScheme={colorScheme}
+          variant="clear"
+          icon={<BiLogOutCircle fontSize="1.5rem" />}
+          aria-label="Sign out"
+        />
+      ) : (
+        <Button
+          colorScheme={colorScheme}
+          variant="clear"
+          rightIcon={<BiLogOutCircle fontSize="1.5rem" />}
+          onClick={onLogout}
+        >
+          {loggedInId} - Sign out
+        </Button>
```

Comment:
seems that we always show this as a button even if `onLogout` is not provided (specifically, we don't provide the `onLogout` prop in the `EndPageView`) 

Should we disable the button in this case? Also, it just seems a bit weird that we allow for "logging out" after the form is submitted anyway. Maybe we should just have something that says "You have been successfully logged out"...

Reply:
disabled button if onLogout is not provided